testing/python/debug/test_lower_trace.py

```
@@ -0,0 +1,727 @@
+
+    env = dict(os.environ)
+    env["TL_LOWER_TRACE"] = "1"
+    env["TL_LOWER_TRACE_DIR"] = str(tmp_path)
+
+    code = (
+        "import tilelang\n"
+        "from tilelang.tools.lower_trace import core as _core\n"
+        "assert _core._is_trace_enabled(), 'tracing not enabled at import time'\n"
+        "assert _core._get_mode() == 'html', 'expected html mode for TL_LOWER_TRACE=1'\n"
+    )
+    result = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)